### ISSUE TITLE
Allow renaming Marines

### ISSUE THREAD

**Issue by GalacticChimp** [Adeptus-Dominus/ChapterMaster#217]
**Description**

Enabling Marine renaming is flavorful, would definitely be nice to have that ability.

**Proposed change**

- Allow renaming Marines (just in Company view should be enough)
- obviously add validation (just check for empty. Players should be able to have Marines named the same, HOWEVER during sector generation names should get duplicated only rarely)

**Comment by GalacticChimp** [comment]
moved to 0.8.3.0 milestone, but it's tentative, we might not have enough time to do implement this, so it might get moved to 0.8.4.0

**Comment by GalacticChimp** [comment]
> marines are already refactored into structs?

ok, I though you were still doing some stuff with them. Ok, then this just got unblocked.

**Comment by GalacticChimp** [comment]
misclicked
